### PR TITLE
HIV intake form

### PR DESCRIPTION
**Pull request by MiguelAHPpih**

### CODE REVIEW

File: configuration/pih/htmlforms/hiv/hiv-intake.xml [hunk 1]
```diff
@@ -171,10 +244,110 @@
             </includeIf>
         </ifMode>
 
+        <section id="elisa" sectionTag="fieldset" headerTag="legend" headerStyle="title" headerCode="pihcore.hivintake.elisa">
+            <div class="section-container">
+                <p>
+                    <obs conceptId ="CIEL:1042"
+                    answerConceptIds="CIEL:703,CIEL:664"
+                    answerLabels="Positivo,Negativo">
+                    </obs>
```
Comment (brandones) [inline configuration/pih/htmlforms/hiv/hiv-intake.xml:253]: Obs tags are self-closing:

```suggestion
                    <obs conceptId ="CIEL:1042"
                    answerConceptIds="CIEL:703,CIEL:664"
                    answerLabels="Positivo,Negativo" />
```

File: configuration/pih/htmlforms/hiv/hiv-intake.xml [hunk 2]
```diff
@@ -171,10 +244,110 @@
...
+                </p>
+            </div>
+        </section>
+
+        <section id="westerblot" sectionTag="fieldset" headerTag="legend" headerStyle="title" headerCode="pihcore.hivintake.wesblot">
+            <div class="section-container">
+                <p>
+                    <obs conceptId ="CIEL:1047"
```
Comment (brandones) [inline configuration/pih/htmlforms/hiv/hiv-intake.xml:261]:
```suggestion
                    <obs conceptId="CIEL:1047"
```

etc.

File: configuration/pih/htmlforms/hiv/hiv-intake.xml [hunk 3]
```diff
@@ -171,10 +244,110 @@
...
+                </p>
+            </div>
+        </section>
+
+        <section id="counseling_hiv" sectionTag="fieldset" headerTag="legend" headerStyle="title" headerCode="pihcore.hivintake.hivCounseling">
+            <div class="section-container">
+                <p>
+                    <obs conceptId ="CIEL:159382"
+                         answerConceptIds="CIEL:1065,CIEL:1066"
+                         answerLabels="Si,No"
```
Comment (brandones) [inline configuration/pih/htmlforms/hiv/hiv-intake.xml:292]:
```suggestion
                         answerLabels="Sí,No"
```

File: configuration/pih/htmlforms/hiv/hiv-intake.xml [hunk 4]
```diff
@@ -171,10 +244,110 @@
...
+                    </obs>
+                </p>
+            </div>
+        </section>
+
+        <section id="tb_comordibity" sectionTag="fieldset" headerTag="legend" headerStyle="title" headerCode="pihcore.hivintake.tbComorbidity">
+            <div class="section-container">
+                <p>
+                    <obs conceptId ="CIEL:42"
+                         answerLabels="Sí,NO"
```
Comment (brandones) [inline configuration/pih/htmlforms/hiv/hiv-intake.xml:316]:
```suggestion
                         answerLabels="Sí,No"
```

Comment (brandones) [reply configuration/pih/htmlforms/hiv/hiv-intake.xml:316]: Is specifying the `answerLabels` necessary? I would expect them to be `Sí,No` by default (per the user's locale).

File: configuration/pih/htmlforms/hiv/hiv-intake.xml [hunk 5]
```diff
@@ -171,10 +244,110 @@
...
+                </p>
+            </div>
+        </section>
+
+        <section id="reg_failed" sectionTag="fieldset" headerTag="legend" headerStyle="title" headerCode="pihcore.hivintake.regimenFailure">
+            <div class="section-container">
+                <p>
+                    <obs conceptId ="PIH:Regimen failed"
+                         answerConceptIds="CIEL:1065,CIEL:1066"
+                         answerLabels="Si,No"
```
Comment (brandones) [inline configuration/pih/htmlforms/hiv/hiv-intake.xml:340]: Same with these `answerLabels`. The concepts, in this case, definitely have the right translations (see [here](https://concepts.pih-emr.org/openmrs/dictionary/concept.htm?conceptId=1065), click "Spanish")

File: configuration/pih/htmlforms/hiv/hiv-intake.xml (hunk 3 above)
Comment (brandones) [reply configuration/pih/htmlforms/hiv/hiv-intake.xml:292]: Actually, see my note below

File: configuration/pih/htmlforms/hiv/hiv-intake.xml (hunk 4 above)
Comment (MiguelAHPpih) [reply configuration/pih/htmlforms/hiv/hiv-intake.xml:316]: I tested and it works like you say. I'll remove the "Sí,No" labels of this form.

File: configuration/pih/htmlforms/hiv/hiv-intake.xml [hunk 6]
```diff
@@ -171,15 +244,110 @@
             </includeIf>
         </ifMode>
 
+        <section id="elisa" sectionTag="fieldset" headerTag="legend" headerStyle="title" headerCode="pihcore.hivintake.elisa">
+            <div class="section-container">
+                <p>
+                    <obs conceptId="CIEL:1042"
+                    answerConceptIds="CIEL:703,CIEL:664"
+                    answerLabels="Positivo,Negativo" />
```
Comment (brandones) [inline configuration/pih/htmlforms/hiv/hiv-intake.xml:252]: These labels are also on the concepts: https://concepts.pih-emr.org/openmrs/dictionary/concept.htm?conceptId=703 under "Spanish"

**Review by brandones — APPROVED** [review]
Looks good! One minor note but feel free to merge when you're happy with it.

**Pull request merged**